HW0/P5/P5.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
```

Comment:
Your graph for P5 is not indicative of the performance trends--both look linear!